W-15895623-scripting-java17-clarification-kt (#2595)
mulesoft/docs-connectors: scripting/2.1/modules/ROOT/pages/scripting-module-upgrade-migrate.adoc

** `<sharedLibraries>` tags under the `<configuration>` section
** `<dependency>` tags under the `<dependencies>` section

[NOTE]

> Starting with Java 17, for your script to execute JavaScript code, it must use GraalVm engine. Therefore, you must add in your `pom.xml` file the following library dependencies: +

Reviewer: ```suggestion
Starting with Java 17, execution of JavaScript code requires the GraalVm engine. To use this engine, add the following library dependencies to your `pom.xml` file : +
```
Take it or leave it. I'm trying to avoid "it must use".
